app/api/hooks.ts

```diff
@@ -289,35 +278,6 @@ export const getUseApiMutation =
       ...options,
     })
 
-/**
- * Our version of `useQueries`, but with the key difference that all queries in
- * a given call are using the same API method, and therefore all have the same
- * request and response (`Params` and `Result`) types. Otherwise the types would
- * be (perhaps literally) impossible.
- */
-export const getUseApiQueries =
-  <A extends ApiClient>(api: A) =>
-  <M extends string & keyof A>(
-    method: M,
-    paramsArray: Params<A[M]>[],
-    options: UseQueryOtherOptions<Result<A[M]>, ApiError> = {}
-  ) => {
-    return useQueries({
-      queries: paramsArray.map(
-        (params) =>
-          ({
-            queryKey: [method, params],
-            queryFn: ({ signal }) =>
-              api[method](params, { signal }).then(handleResult(method)),
-            throwOnError: (err: ApiError) => err.statusCode === 404,
-            ...options,
-            // Add params to the result for reassembly after the queries are returned
-            select: (data) => ({ ...data, params }),
-          }) satisfies UseQueryOptions<Result<A[M]> & { params: Params<A[M]> }, ApiError>
-      ),
-    })
-  }
-
```

Comment:
This (and `useApiQueries`) wasn't used. I kept it around in case we needed it again, but I think if we wanted this now, we'd use the query options helper with `map` and pass the resulting array of options objects to the built-in `useQueries`.